angular/src/app/discovery/beacon/beacon-search.component.html

```diff
@@ -102,6 +101,7 @@ <h2>Beacon</h2>
                     <span>{{ datasetAlleleResponse.datasetId || datasetAlleleResponse['id'] }}</span>
                   </ng-template>
                   <ddap-beacon-search-result-table (selectedRowChanged)="selectedRowData = $event"
+                                                   (rowClicked)="selectedRowDetailDrawer.open()"
```

Comment:
this can be chained into `(selectedRowChanged)="selectedRowData = $event; selectedRowDetailDrawer.open()"` or create a new function for it, whatever you think is better.